Library/Homebrew/rubocops/cask/ast/stanza.rb

```diff
@@ -12,12 +12,12 @@ module AST
       class Stanza
         extend Forwardable
 
-        def initialize(method_node, comments)
+        def initialize(method_node, cask_node)
           @method_node = method_node
```

Comment:
```suggestion
          @stanza_node = method_node
```

so we don't need the `alias` below.

Reply:
Nevermind, I think we need to keep `method_node` since other nodes use the same naming.